### PR TITLE
Dev stack update

### PR DESCRIPTION
**Pull request by ashleysommer**
Initial python 3.7 requirement update, add new Drone tests, add black.toml file, add .editorconfig file

~Draft PR, do not merge yet,~ testing Drone compatibility with Black

### CODE REVIEW

**Comment by ashleysommer** [comment]
tests passed, fee free to merge after review.

**Comment by ashleysommer** [comment]
TODO: After this is merged, re-black all /rdflib/*.py files with the new recommended Black version with the given black.toml config file.

File: .drone.yml [hunk 1]
```diff
@@ -70,6 +49,28 @@ steps:
   - pip install --default-timeout 60 -r requirements.dev.txt
   - pip install --default-timeout 60 coveralls && export HAS_COVERALLS=1
   - python setup.py install
```
Comment (ashleysommer) [inline .drone.yml:51]:
```suggestion
  - pip install --default-timeout 60 -r requirements.dev.txt
  - python setup.py install
```

File: .drone.yml [hunk 2]
```diff
@@ -70,6 +49,28 @@ steps:
...
+
+steps:
+- name: test
+  image: python:3.9
+  commands:
+  - bash .travis.fuseki_install_optional.sh
+  - pip install --default-timeout 60 -r requirements.txt
+  - pip install --default-timeout 60 -r requirements.dev.txt
+  - pip install --default-timeout 60 coveralls && export HAS_COVERALLS=1
+  - python setup.py install
```
Comment (ashleysommer) [inline .drone.yml:72]:
```suggestion
  - pip install --default-timeout 60 -r requirements.dev.txt
  - python setup.py install
```

**Review by edmondchuc — APPROVED** [review]

**Pull request merged**